e2e/e2e_unbond_test.go

```diff
@@ -0,0 +1,81 @@
+package e2e
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdktypes "github.com/cosmos/cosmos-sdk/types"
+	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
+)
+
+func (s *IntegrationTestSuite) testUnbond() {
+	s.Run("unbond", func() {
+		val1Addr, err := s.chain.validators[0].keyInfo.GetAddress()
+		s.Require().NoError(err)
+		val1 := val1Addr.String()
+
+		val2Addr, err := s.chain.validators[1].keyInfo.GetAddress()
+		s.Require().NoError(err)
+		val2 := val2Addr.String()
+
+		s.execUnbond(s.chain, 0, sdk.ValAddress(val1Addr), val1, standardFees.String(), false)
+		s.Require().Eventually(
+			func() bool {
+				batch, err := queryBatch(s.endpoint, 1)
+				s.Require().NoError(err)
+				return len(batch.BatchSignatures) == len(s.chain.validators)
+			},
+			30*time.Second,
+			5*time.Second,
+		)
+
+		s.execUnbond(s.chain, 1, sdk.ValAddress(val2Addr), val2, standardFees.String(), false)
```

Comment:
I'm a little confused how this can ever work? The E2E chain only has 2 validators right? So when we unbond the 2nd one we should not get any new blocks.

Reply:
Never mind I didn't see that it only unbonds a tiny amount